extensions.json

```diff
@@ -167,6 +171,18 @@
             "archive": "https://github.com/lnbits/tpos/archive/refs/tags/v0.7.1.zip",
             "hash": "559670df0055af8d99419910739c8906ad9b1bf41d88660452990d33e0dd17c5"
         },
+        {
+            "id": "tpos",
+            "repo": "https://github.com/lnbits/tpos",
+            "name": "TPoS",
+            "version": "0.7.2",
```

Comment:
there is no `0.7.2` release for TPOS

Reply:
I know, weird, did you add?

Reply:
fixed now

Reply:
https://github.com/lnbits/lnbits-extensions/pull/374/commits/f3590764f767f6da62f41baf317b32195bc1cdc1